### DISCUSSION TITLE
update package version along with dependencies

### DISCUSSION THREAD

**Discussion by mikethecalamity** [renovatebot/renovate#29616] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab

### Please tell us more about your question or problem

Is it possible to update the package version along with the dependencies in a `package.json`?

```json
{
  "name": "my-project",
  "version": "16.5.0", <-------------- THIS
  "description": "test ",
  "author": "mikethecalamity",
  "dependencies": {
    "lodash": "^4.17.21",
    "node-fetch": "^2.6.1"
  },
  "scripts": {
    "lint": "node node_modules/eslint src/**.js",
    "build": "node build.js"
  }
}
```

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Yes, see bumpVersion
